cinema/models.py

```diff
@@ -1,6 +1,10 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.conf import settings
+from django.utils.text import slugify
```

Comment:
sort imports

Reply:
@BogdanFSD not fixed
PEP8 imports order